jutut/local_settings.example.py

```diff
@@ -8,6 +8,9 @@
 #SERVER_EMAIL = 'root@example.com' # Error message sender
 #DEFAULT_FROM_EMAIL = 'webmaster@example.com' # Other email sender
 
+# Make sure this points to the correct virtual environment location
+STATICFILES_DIRS += join(BASE_DIR, '../venv/js_jquery_toggle/static'),
```

Comment:
This shouldn't be required. The static file is found as it's part of the js_jquery_toggle django app. Ensure that:
1. You have installed the django wrapped version of js_jquery_toggle
2. `INSTALLED_APPS` includes the django part of the js_jquery_toggle (it's possible it was included as a dependency using raphendyr-django-essentials featyre, I don't remember)
3. Ensure the js_jquery_toggles, setup.py installs static files correctly (packages them to the wheel and then installs those to correct paths)

Reply:
For the future, wrapping js libraries on django apps isn't a good idea, instead they should be pulled as part of the CI/CD pipeline and packaging (see A+ drone configs). However, none of the Django apps in the project include packaging :cry:

Reply:
Are you using a version, which is after this commit: https://github.com/Aalto-LeTech/js-jquery-toggle/commit/e100529371a6e3dba2fc22a6a57fc80f4d1ededf

Reply:
Yes, I think that is the issue. Update the requirements.txt with the 1.1.1 version of the lib.

```patch
-git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.0#egg=js-jquery-toggle-django==1.1.0&subdirectory=django
+git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2#egg=js-jquery-toggle-django==1.1.2&subdirectory=django
```

Reply:
That's definitely a lot more elegant solution!